src/components/Badge/Badge.docs.mdx

```diff
@@ -0,0 +1,46 @@
+import { Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import Badge from './Badge';
+
+# Badge
+
+A `Badge` is a round image of predetermined size.
```

Comment:
Could you include some examples of when you may and may not want to use this component? E.g. yes for avatars, no for editorial images